constitution.md

```diff
@@ -0,0 +1,684 @@
+
+* conducting and publishing research into improvements to the processes used in open source and related technologies,
+* working with government at all levels to ensure that the interests of the open source and related technologies are represented in regard to the public decision-making process, and
+* providing a forum for all people engaged in the open source and related technologies to discuss best practice and enhancing the future of the software and computing.
+
+### _PreObj_: Objects of the organisation
+
+_PreObj-1_: The objects for which the association is established are:
+
+* to facilitate the communication of information relating to inventions, results, techniques, and governance of open source and related open technologies;
+* to support open source and related open technologies projects and the contributors to open source and related open technologies;
+* to support and promote the adoption of open source and related open technologies;
+* to support and promote awareness and understanding of open source and related open technologies;
+* to conduct open source and related open technologies advocacy with the government, business and civil society;
+* to create a network for related local and online open source and related open technologies communities.
+
+# Part 2 - Members of association
+
+### _MemGen_: Membership generally
+
+_MemGen-1_: An individual is taken to be a member of the association if:
+
+* the person applied to be a member under clause _MemApp-1_ and the application has been approved, or
+* the person was 1 of the individuals on whose behalf an application for registration of the association was made under the Act, section 6(1)(a), or
+* the person was:
+  - for an unincorporated body registered as the association - a member of the body immediately before the registration, or
+  - for a registrable corporation registered as the association - a member of the corporation immediately before the registration, or
+  - for an association that was amalgamated to form the relevant association - a member of the existing association immediately before the amalgamation.
+
+_MemGen-2_: A person who is not an individual is not eligible to be a member of the association.
+
+### _MemApp_: Membership applications
+
+_MemApp-1_: An application by a person to be a member of the association must be:
+
+- made in writing, and
+- in the form determined by the committee, and
+- lodged with the secretary.
+
+_MemApp-2_: The committee may determine that an application may be made or lodged by email or other electronic means.
+
+_MemApp-3_: The secretary must determine whether to approve or reject the application or refer the nomination to the committee which is to determine whether to approve or to reject the nomination.
+
+_MemApp-4_: The committee must approve or reject the application.
+
+_MemApp-5_: As soon as practicable after the committee has decided the application, the secretary must:
```

Comment:
There is some inconsistency here introduced by a4d73c5. The application may be decided by the secretary, rather than the committee.

Reply:
> There is some inconsistency here introduced by [a4d73c5](https://github.com/linuxaustralia/constitution_and_policies/commit/a4d73c5d9476a3bce33e13174a2654889f2eedf3). The application may be decided by the secretary, rather than the committee.

Agreed.  I will fix this in the next commit.